### PR TITLE
Update support_00_local_disc.md

### PR DESCRIPTION
**Pull request by korbinib**
Added:
* NALMIN
* NAPI
* NOR-Openscreen

### CODE REVIEW

File: pages/dmp_supporting_info/support_00_local_disc.md [hunk 1]
```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
...
+
 ### NorCRIN - Clinical research
 The national infrastructure NorCRIN provides advice on national procedures, good clinical practice, data management and monitoring. NorCRIN is a partnership between six university hospitals in Norway with the objective to strengthen and simplify collaboration in all categories of clinical research in Norway and member of the [European Clinical Research Infrastructure Network (ECRIN ERIC)](https://ecrin.org/).
 
 Homepage: [NorCRIN](https://www.norcrin.no/ "2024-07-12")
 
 Knowledge resource: [Guidance and support (NorCRIN): Are you planning a clinical study?](https://www.norcrin.no/en/are-you-planning-a-clinical-study/)
 
+### NOR-Openscreen - Chemical biology
+NOR-Openscreen is the Norwegian node of the [EU-Openscreen infrastructure (EU-Openscreen ERIC)](https://www.eu-openscreen.eu/). We support the discovery of biologically active substances in all areas of the Life Sciences by providing transnational, open access to the most advanced technologies, chemical and biological resources as well as expertise through Europe.
```
Comment (jennyostrop) [inline pages/dmp_supporting_info/support_00_local_disc.md:99]:
```suggestion
NOR-Openscreen is the Norwegian node of the [EU-Openscreen infrastructure (EU-Openscreen ERIC)](https://www.eu-openscreen.eu/) supporting the discovery of biologically active substances in all areas of the Life Sciences by providing transnational, open access to the most advanced technologies, chemical and biological resources as well as expertise through Europe. 
```

File: pages/dmp_supporting_info/support_00_local_disc.md [hunk 2]
```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
...
+
+Homepage: [NOR-Openscreen](https://www.openscreen.uio.no/english/index.html "2024-08-12")
+
+Knowledge resource: [NOR-Openscreen offers guidance about FAIR data management](https://www.openscreen.uio.no/english/services/data-management/index.html "2024-08-12")
```
Comment (jennyostrop) [inline pages/dmp_supporting_info/support_00_local_disc.md:103]: Rather helpdesk/contact? Little information on this page.

**Review by jennyostrop — CHANGES_REQUESTED** [review]

File: pages/dmp_supporting_info/support_00_local_disc.md (hunk 2 above)
Comment (korbinib) [reply pages/dmp_supporting_info/support_00_local_disc.md:103]: I can add the general contact - but it's not only for RDM

```suggestion
Knowledge resource: [NOR-Openscreen offers guidance about FAIR data management](https://www.openscreen.uio.no/english/services/data-management/index.html "2024-08-12")

Helpdesk:  [chembio@ncmm.uio.no](mailto:chembio@ncmm.uio.no?subject=Research Data Management)
```

File: pages/dmp_supporting_info/support_00_local_disc.md [hunk 3]
```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
...
+
 ### NorCRIN - Clinical research
 The national infrastructure NorCRIN provides advice on national procedures, good clinical practice, data management and monitoring. NorCRIN is a partnership between six university hospitals in Norway with the objective to strengthen and simplify collaboration in all categories of clinical research in Norway and member of the [European Clinical Research Infrastructure Network (ECRIN ERIC)](https://ecrin.org/).
 
 Homepage: [NorCRIN](https://www.norcrin.no/ "2024-07-12")
 
 Knowledge resource: [Guidance and support (NorCRIN): Are you planning a clinical study?](https://www.norcrin.no/en/are-you-planning-a-clinical-study/)
 
+### NOR-Openscreen - Chemical biology
+NOR-Openscreen is the Norwegian node of the [EU-Openscreen infrastructure (EU-Openscreen ERIC)](https://www.eu-openscreen.eu/) supporting the discovery of biologically active substances in all areas of the Life Sciences by providing transnational, open access to the most advanced technologies, chemical and biological resources as well as expertise through Europe.
```
Comment (jennyostrop) [inline pages/dmp_supporting_info/support_00_local_disc.md:99]:
```suggestion
NOR-Openscreen is the Norwegian node of the [EU-Openscreen infrastructure (EU-Openscreen ERIC)](https://www.eu-openscreen.eu/) supporting the discovery of biologically active substances in all areas of the Life Sciences by providing transnational, open access to the most advanced technologies, chemical and biological resources as well as expertise through Europe. NOR-Openscreen offers guidance about FAIR data management.
```

File: pages/dmp_supporting_info/support_00_local_disc.md [hunk 4]
```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
...
 Homepage: [NorCRIN](https://www.norcrin.no/ "2024-07-12")
 
 Knowledge resource: [Guidance and support (NorCRIN): Are you planning a clinical study?](https://www.norcrin.no/en/are-you-planning-a-clinical-study/)
 
+### NOR-Openscreen - Chemical biology
+NOR-Openscreen is the Norwegian node of the [EU-Openscreen infrastructure (EU-Openscreen ERIC)](https://www.eu-openscreen.eu/) supporting the discovery of biologically active substances in all areas of the Life Sciences by providing transnational, open access to the most advanced technologies, chemical and biological resources as well as expertise through Europe. 
+
+Homepage: [NOR-Openscreen](https://www.openscreen.uio.no/english/index.html "2024-08-12")
+
+Helpdesk:  [chembio@ncmm.uio.no](mailto:chembio@ncmm.uio.no?subject=Research Data Management)
```
Comment (jennyostrop) [inline pages/dmp_supporting_info/support_00_local_disc.md:103]:
```suggestion
Helpdesk: [chembio@ncmm.uio.no](mailto:chembio@ncmm.uio.no?subject=Research Data Management)
```

File: pages/dmp_supporting_info/support_00_local_disc.md [hunk 5]
```diff
@@ -78,13 +78,30 @@ Homepage: [GBIF Norway](https://www.gbif.no/ "2024-07-19")
 
 Helpdesk: [helpdesk@gbif.no](https://www.gbif.no/services/ "2024-07-19")
 
+### NALMIN - Norwegian Advanced Light Microscopy Imaging Network
+Norwegian Advanced Light Microscopy Imaging Network (NALMIN) principal goal is to establish a network at international competence level for advanced light microscopy. The network consists of 5 national nodes located in Oslo (1 node with two branches(NorMIC IBV and NorMIC Radium)), Bergen, Trondheim (2 nodes) and Tromsø, each with specialized competence. All nodes are members of the national NorBioimaging infrastructure organization, and the NorMIC Oslo node is a member of the [EuroBioimaging ECRIN](https://www.eurobioimaging.eu/nodes/normic-oslo---advanced-light-microscopy-node-oslo) network.
```
Comment (jennyostrop) [inline pages/dmp_supporting_info/support_00_local_disc.md:82]: Are the node specifics really necessary?

**Review by jennyostrop — CHANGES_REQUESTED** [review]

File: pages/dmp_supporting_info/support_00_local_disc.md (hunk 5 above)
Comment (korbinib) [reply pages/dmp_supporting_info/support_00_local_disc.md:82]: I think this is helpful, since most of the support is provided locally.

**Review by jennyostrop — APPROVED** [review]

**Pull request merged**